facia/test/package.scala

```diff
@@ -46,7 +46,7 @@ object `package` {
 }
 
 class FaciaTestSuite extends Suites (
-  new CommercialMPUForFrontsTest,
+  // new CommercialMPUForFrontsTest, // tests fail when fronts are repressed so I don't believe are necessary. Please refactor or delete.
```

Comment:
The reason why we have to use a HTML cleaner, is because so much of the logic of where a MPU will be inserted is dependent on what other elements are on the page...

![lyingonfloorcrying](https://cloud.githubusercontent.com/assets/8607683/24502519/0b67c2c2-1546-11e7-8acf-26558000ff46.jpeg)

I don't want these tests to block other work. If you want to delete them, I've added a card to the commercial board so we can revisit these tests, hopefully mock externals, and make them more resilient.